tests/cross/src/test/scala/tests/hover/HoverTermSuite.scala

```diff
@@ -240,11 +250,17 @@ class HoverTermSuite extends BaseHoverSuite {
       |""".stripMargin,
     """|Option[String]
        |def flatMap[B](f: Int => Option[B]): Option[B]
-       |""".stripMargin.hover
+       |""".stripMargin.hover,
+    compat = Map(
+      "3" ->
+        """|Boolean
+           |def withFilter(p: A => Boolean): Option.this.WithFilter
```

Comment:
Maybe this is what Scala3 actually transformed to? I'll check the tree after typer by Scala3 compiler 🤔

Reply:
Looks like this should be shown in the line underneath

Reply:
Can we address this in another PR?

We have something like,

```scala
Option.apply[Int](1).withFilter(...).flatMap(...)
```

 but `Interactive.pathTo()` returns the path to `withFilter` instead of `flatMap`. Maybe we need a fix in `Interacitve.pathTo` side?